experimental/PropertyDDS/packages/property-inspector-table/src/ExpiryModal.tsx

```diff
@@ -374,7 +374,7 @@ export const ExpiryModal: React.FunctionComponent<IExpiryModalProps> = (props) =
     return (
       <div className={classes.contentContainer}>
         Are you sure you want to delete this repository?<br />
-        By deleting it, the Lifecycle State will change to 'Expired' for 30 days before being destroyed.
+        By deleting it, the Lifecycle State will change to &quot;Expired&quot; for 30 days before being destroyed.
```

Comment:
Nit: this does change the emitted HTML from single to double quotes.

Reply:
I chose to do that intentionally since it's standard English to use double-quotes in this context.